### ISSUE TITLE
Sdl2 pkg-config missing 

### ISSUE THREAD

**Issue by AntonioCS** [Alexpux/MINGW-packages#3261]
Hey,

Not sure if this is the correct place but here it goes.
I recently installed some SDL2 packages:
```

mingw64/mingw-w64-x86_64-SDL2 2.0.7-1 [installed]
    A library for portable low-level access to a video framebuffer, audio output, mouse, and keyboard (Version 2) (mingw-w64)
mingw64/mingw-w64-x86_64-SDL2_gfx 1.0.1-2 [installed]
    SDL graphics drawing primitives and other support functions wrapped up in an addon library for the Simple Direct Media (SDL) cross-platform API layer
mingw64/mingw-w64-x86_64-SDL2_image 2.0.2-1 [installed]
    A simple library to load images of various formats as SDL surfaces (Version 2)
mingw64/mingw-w64-x86_64-SDL2_mixer 2.0.2-2 [installed]
    A simple multi-channel audio mixer (Version 2) (mingw-w64)
mingw64/mingw-w64-x86_64-SDL2_net 2.0.1-1 [installed]
    A small sample cross-platform networking library (Version 2) (mingw-w64)
mingw64/mingw-w64-x86_64-SDL2_ttf 2.0.14-1 [installed]
```

I would expect this command:
pkg-config --libs --cflags sdl2

To output the required data but what I get is this:
```

$ pkg-config --libs --cflags sdl2
Package sdl2 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sdl2.pc'
to the PKG_CONFIG_PATH environment variable
No package 'sdl2' found
```

I remember doing this in past installations and getting what I wanted from pkg-config.
Am I missing something here? Is something broken?

**Comment by peterbud** [comment]
I guess you were using the `MSYS` shell . Use the `MINGW64` shell:
```
$ pkg-config --libs --cflags sdl2

-Dmain=SDL_main -IC:/msys64/mingw64/include/SDL2 -LC:/msys64/mingw64/lib -lmingw32 -lSDL2main -lSDL2 -mwindows
```

**Comment by AntonioCS** [comment]
I am just using a windows terminal. I setup msys2 in the PATH variables but I might have added the wrong path. Will update and will let you know.
Thanks!

**Comment by AntonioCS** [comment]
In my path variables I have this:

```
C:\msys64\usr\bin
C:\msys64\mingw64\bin
```

So I am pointing to mingw64, right? Should I have other paths here?

**Comment by Alexpux** [comment]
Mingw64 patch need to be before usr. Also you need set PKG_CONFIG_PATH variable maybe

**Comment by AntonioCS** [comment]
Yeah... that didn't work out. This is strange, I had these exact settings in another machine and this worked perfectly.
I can see it work just fine in the Msys2 console, so I am not really sure what to do next. 
I will try to see if I have anything different on the other machine.

**Comment by AntonioCS** [comment]
A quick note, in my other machine I noticed that I have both versions of SDL2 installed:

```
> pacman -Ss sdl2
mingw32/mingw-w64-i686-SDL2 2.0.5-1 [installed]
    A library for portable low-level access to a video framebuffer, audio output, mouse, and keyboard (Version 2) (mingw-w64)
mingw32/mingw-w64-i686-SDL2_gfx 1.0.1-2
    SDL graphics drawing primitives and other support functions wrapped up in an addon library for the Simple Direct Media (SDL) cross-platform API layer
mingw32/mingw-w64-i686-SDL2_image 2.0.1-2 [installed]
    A simple library to load images of various formats as SDL surfaces (Version 2)
mingw32/mingw-w64-i686-SDL2_mixer 2.0.1-3 [installed]
    A simple multi-channel audio mixer (Version 2) (mingw-w64)
mingw32/mingw-w64-i686-SDL2_net 2.0.1-1
    A small sample cross-platform networking library (Version 2) (mingw-w64)
mingw32/mingw-w64-i686-SDL2_ttf 2.0.14-1 [installed]
    A library that allows you to use TrueType fonts in your SDL applications (Version 2) (mingw-w64)
mingw32/mingw-w64-i686-smpeg2 2.0.0-4
    SDL2 MPEG Player Library (mingw-w64)
mingw64/mingw-w64-x86_64-SDL2 2.0.5-1 [installed]
    A library for portable low-level access to a video framebuffer, audio output, mouse, and keyboard (Version 2) (mingw-w64)
mingw64/mingw-w64-x86_64-SDL2_gfx 1.0.1-2 [installed]
    SDL graphics drawing primitives and other support functions wrapped up in an addon library for the Simple Direct Media (SDL) cross-platform API layer
mingw64/mingw-w64-x86_64-SDL2_image 2.0.1-2 [installed]
    A simple library to load images of various formats as SDL surfaces (Version 2)
mingw64/mingw-w64-x86_64-SDL2_mixer 2.0.1-3 [installed]
    A simple multi-channel audio mixer (Version 2) (mingw-w64)
mingw64/mingw-w64-x86_64-SDL2_net 2.0.1-1 [installed]
    A small sample cross-platform networking library (Version 2) (mingw-w64)
mingw64/mingw-w64-x86_64-SDL2_ttf 2.0.14-1 [installed]
    A library that allows you to use TrueType fonts in your SDL applications (Version 2) (mingw-w64)
mingw64/mingw-w64-x86_64-smpeg2 2.0.0-4 [installed]
    SDL2 MPEG Player Library (mingw-w64)

```

Now when I do: pkg-config --libs --cflags sdl2
I get:

`-Dmain=SDL_main -IC:/msys64/mingw64/include/SDL2 -LC:/msys64/mingw64/lib -lmingw32 -lSDL2main -lSDL2

`

They seem to be the 64bit versions of the libs. Can anyone tell me what else I can do?

**Comment by peterbud** [comment]
I'd suggest to read the [wiki](https://github.com/msys2/msys2/wiki) first, and if you still have questions then feel free to ask.

**Comment by AntonioCS** [comment]
@peterbud Any specific part of the wiki that I should read? 
Am I suppose to use the msys2 shell for everything? I was really hoping I could just use a normal terminal windows to get what I need.

**Comment by peterbud** [comment]
No, a windows command prompt window won't work.

@AntonioCS I'd not recommend to do any development until you have time to read the wiki as a bare minimum - it would be simply a waste of your (and probably other people's) time. I'm not sure what you would like to achieve, but if you feel reading 20 pages is too much time investment, I'd suggest to consider some other development platform/tool with lower entry point.

**Comment by AntonioCS** [comment]
What I would like to achieve is executing that command and getting the correct output. 

I have read the wiki and that is why I am asking for a specific part of the wiki. Asking if there is something I missed or possible misread.

I have also demonstrated that I have made this work in another machine but possibly not the exact libs I wanted.
But as I can see I won't get any help from here I will close this and you go patronise someone else.
